npc/quests/quests_lighthalzen.txt

```diff
@@ -4206,7 +4207,7 @@ lhz_que01,99,74,5	script	#li_Man	48,{
 	end;
 }
 
-lhz_que01,90,71,5	script	#li_bird	1019,3,3,{
+lhz_que01,90,71,5	script	#li_bird	PECOPECO,3,3,{
```

Comment:
You can leave this as `1019` since we don't export mob names (they are technically defined in the mob_db.

Reply:
Change made https://github.com/rathena/rathena/pull/4392